frontend/src/components/CreateEventForm/CreateEventForm.tsx

```diff
@@ -0,0 +1,223 @@
+                        <input
+                          type='checkbox'
+                          value={connection.id}
+                          onChange={e => {
+                            if (e.target.checked) {
+                              setSelectedInvites(prev => [...prev, connection.id]);
+                            } else {
+                              setSelectedInvites(prev => prev.filter(id => id !== connection.id));
+                            }
+                          }}
+                        />
+                        <span className='text-sm'>
+                          {connection.name} (@{connection.twitchUser})
+                        </span>
+                      </label>
+                    ))
+                  ) : (
+                    <p className='text-sm text-gray-500'>No connections to invite</p>
+                  )}
+                </div>
+                {/*Nice user experience to include how many people are being invited */}
+                <p className='text-xs text-gray-500'>
+                  {selectedInvites.length} connection(s) selected to be invited
+                </p>
+              </div>
+            )}
+
+            <div className='flex flex-col gap-2'>
+              <Label htmlFor='event_modality'>Event Modality</Label>
+              <Select name='event_modality' onValueChange={setEventModality} required>
+                <SelectTrigger className='w-full'>
+                  <SelectValue placeholder='Select your events modality' />
+                </SelectTrigger>
+                <SelectContent>
+                  {EVENT_MODALITIES.map(eventModality => (
+                    <SelectItem id={eventModality.id} value={eventModality.id}>
+                      {eventModality.label}
+                    </SelectItem>
+                  ))}
+                </SelectContent>
+              </Select>
+            </div>
+
+            {/*If user specifies that their event is in person we want them to be able to enter the location its taking place at */}
+            {eventModality === "in_person" && (
+              <div className='flex flex-col gap-2'>
+                <Label htmlFor='location'>Location</Label>
+                <Input
+                  id='location'
+                  name='location'
+                  placeholder='Enter event location...'
+                  required
+                />
+              </div>
+            )}
+
+            <div className='flex flex-col gap-2'>
+              <Label htmlFor='tags'>Event Tags</Label>
+              <Input id='tags' name='tags' placeholder='valorant, fun, soccer' required />
+              <p className='text-muted-foreground mt-1 text-sm'>
+                Separate tags with commas (e.g., fun, soccer, chatting)
+              </p>
+            </div>
+
+            <Button type='submit' className='bg-electric-indigo cursor-pointer'>
```

Comment:
Should you disable this if the event is private and there are no connections selected?